Configuration.hpp

```diff
@@ -32,22 +32,22 @@
 // Include the current software version.
 #include "Version.h"
 
-// Include the user-specific local configuration
-#if __has_include("Configuration_local.hpp")
-    #include "Configuration_local.hpp"
-#elif __has_include("Configuration_local_CI.hpp")                                   // CI environment on GitHub
-    #include "Configuration_local_CI.hpp"
-#endif
-
 /**
  * Use default values for any parameters the user didn't provide.
  */
 
 // If board is not defined in local configuration, assume Arduino Mega2560
 #ifndef BOARD
-  #define BOARD BOARD_AVR_MEGA2560
+  // #define BOARD BOARD_AVR_MEGA2560
```

Comment:
May as well delete these three lines if you're going to error out right after anyway.

Reply:
My idea was to provide an example here this way (in case people dont use local config files)

Reply:
but i see now the comment does not match now :)

Reply:
I have adapted the code comment. In general we need this definition because Arduino IDE is not able to set it. But with new comment it should be hopefully clear when and how to set it.